pkg/transport/http.go

```diff
@@ -162,11 +174,13 @@ func (t *SSETransport) Setup(ctx context.Context, runtime rt.Runtime, containerN
 	t.containerID = containerID
 	logger.Infof("Container created with ID: %s", containerID)
 
-	// If the SSEHeadlessServiceName is set, use it as the target host
-	// This is useful for Kubernetes deployments where the workload is
-	// exposed as a headless service.
-	if containerOptions.SSEHeadlessServiceName != "" {
-		t.targetHost = containerOptions.SSEHeadlessServiceName
+	if t.Mode() == types.TransportTypeSSE {
```

Comment:
Do we need something similar for streamable HTTP?

Reply:
i don't think so. Reading documentation i did not find the need of overriding service name